kernel/src/main/java/org/kframework/parser/concrete2kore/ParserUtils.java

```diff
@@ -36,7 +36,7 @@
     public ParserUtils(FileUtil files) {
         this.files = files;
     }
-    public static K parseWithFile(CharSequence theTextToParse,
+    public static K parseWithFile(String theTextToParse,
```

Comment:
Because I was using the `run` method, which takes as input a `String`. (see continuation bellow)